server/sonar-web/src/main/js/apps/component-measures/sidebar/DomainFacet.js

```diff
@@ -67,9 +67,10 @@ export default class DomainFacet extends React.PureComponent {
     const { domain, selected } = this.props;
     const measureSelected = domain.measures.find(measure => measure.metric.key === selected);
     const overviewSelected = domain.name === selected && hasBubbleChart(domain.name);
-    return measureSelected
-      ? [getLocalizedMetricName(measureSelected.metric)]
-      : overviewSelected ? [translate('component_measures.domain_overview')] : [];
+    if (measureSelected) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add curly braces around the nested statement(s) in this "if" block. ([javascript:CurlyBraces](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ACurlyBraces))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fpm%2Ffix-sonar-10452&issues=AWH6-c6ma5Ltb7kATvqy&open=AWH6-c6ma5Ltb7kATvqy)

[ISSUE_KEY]: AWH6-c6ma5Ltb7kATvqy